### PR TITLE
Chaplain Buffs

### PR DESCRIPTION
**Pull request by Kaedwuff**
-Adds an aspergillum, which will hold up to 10u of liquids, such as holy water, and spray out out to adjacent squares, such as those occupied by filthy vampires.  It also makes a pretty jingle sound when you do so. The sprite was provided by Dronz.
-Removes the buggy null item box, and replaces it with obsidian shards that can be reassembled at will to whatever null item shape you crave.
-Null items can now be mounted on poncho backs
-Some backend updates to spraybottle code that will let you change the sound effect of a spray-object if someone makes one in the future.

### CODE REVIEW

**Comment by VTCobaltblood** [comment]
Changelog's wrong. Also, how about not using others' work when they clearly don't feel comfortable with it?

**Comment by Kaedwuff** [comment]
They provided me with it ages ago for this update, and in absence of a better sprite, it will suffice.

**Comment by VTCobaltblood** [comment]
It's for the wrong branch, apparently. Only the poncho thing is listed.

**Comment by Kaedwuff** [comment]
That was fixed literally in the update I posted before you did.

**Comment by alsoandanswer** [comment]
For lack of a better sprite, you can always request for one. Don't be afraid to ask.

**Comment by Kaedwuff** [comment]
Do you have a suggestion? I'm not sure where I would ask.

**Comment by Kaedwuff** [comment]
!review

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	throw_range = 4
 	throwforce = 10
 	w_class = 2
+	var/global/list/nullchoices
```
Comment (Arrow768) [inline code/game/objects/items/weapons/weaponry.dm]: Why does that need to be a global var?

Comment (Kaedwuff) [reply code/game/objects/items/weapons/weaponry.dm]: This is a modified version of the code used for chameleon items. So, I guess for the same reason it's global there? Does it need to be changed?

Comment (Sindorman) [reply code/game/objects/items/weapons/weaponry.dm]: not really, it only needs to be global if some other things in other file is trying access it. But then they can just access var from the object itself. So I see no reason for it to be global

Comment (Lohikar) [reply code/game/objects/items/weapons/weaponry.dm]: There is no reason to make each instance regenerate this list as it's always the same. This is global so it only gets generated once.

Comment (skull132) [reply code/game/objects/items/weapons/weaponry.dm]: Wouldn't it be better noted as `static` then? Granted, in DM the difference might well be semantics. But the behaviour of this matches more closely what the classical definition for a class static variable is.

Comment (Lohikar) [reply code/game/objects/items/weapons/weaponry.dm]: `static` would probably be clearer, yes.

**Comment by Kaedwuff** [comment]
So what's the sitrep on this being released?

**Pull request merged**